src/GameObjects/Player.py

```diff
@@ -1,17 +1,26 @@
 import pygame
 
-from Interface.GameObjectInterface import GameObject
+from GameObjects.Interface.GameObjectInterface import GameObject
 
 
 class Player(GameObject):
-    def __init__(self, x, y):
+    def __init__(self, x, y, width, height):
         self.x = x
         self.y = y
+        self.width = width
+        self.height = height
         # Modify the hitbox size depending on the Image size for player
-        self.hitbox = (self.x, self.y, 20, 20)
+        self.hitbox = (self.x, self.y,self.width, self.height)
 
     def drawCharacter(self, canvas):
-        pass
+        pygame.draw.rect(canvas,(0,255,0),self.hitbox)
+        self.hitbox = (self.x, self.y, self.width, self.height)
+
+    def propagate(self, canvas, step):
+        self.drawCharacter(canvas)
+        keys = pygame.key.get_pressed()
+        if keys[pygame.K_LEFT] and self.x > step:
+            self.x -= step
+        elif keys[pygame.K_RIGHT] and self.x < canvas.get_width() - self.width - step:
+            self.x += step
```

Comment:
This is not the correct place to define key_events..
This would mean that we need to call propagate in every frame..
I would suggest that we define key_events in Game.py foler and call propagate accordingly.